File: Programacion/UD4/ejercicios1 tema4/src/ejercicio4.java
import java.util.Scanner;

public class ejercicio4 {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.println("Dime un numero");
        int num = sc.nextInt();

        etm(num);

    }

    static void etm(int num) {

        for (int i = 1; i <= 10; i++){

            System.out.println(num + "*" + i + "=" + (num*i));


        }


    }
}

File: Programacion/UD7/EjerciciosKotlin1/src/Ejercicio4.kt
import java.util.Scanner
import kotlin.math.sqrt

fun esPrimo(num: Int): Boolean{
    if (num <= 1) {
        return false
    } else {
        for (i in 2..sqrt(num.toDouble()).toInt()){
            if (num % i == 0){
                return false
            }
        }
    }
    return true
}

fun listaNumerosPrimos(num: Int){
    for (a in 2..num){
        var es: Boolean = true
        for (i in 2..sqrt(num.toDouble()).toInt()){
                if (a % i == 0){
                    es = false
                }
        }
        if (es) {
            print("$a ")
        }

    }

}



fun main() {
    var sc = Scanner(System.`in`)
    println("Dime el numero a verificar si es primo")
    var numero = sc.nextInt()
    if (esPrimo(numero)){
        println("Es primo")
    } else{
        println("No es primo")
    }
    println("Ahora pongo la lista de primos hasta el numero indicado")
    listaNumerosPrimos(numero)

}